job-service-contract/src/main/resources/com/hpe/caf/services/job/swagger.yaml

```diff
@@ -335,6 +337,41 @@ definitions:
         example:
           "tag:4": "4"
           owner: "bob"
+  expirationPolicy:
+    type: object
+    properties:
```

Comment:
Aren't you introducing a new Expired status, so shouldn't it be listed here?  Is it special since the `expire` action makes no sense for it?

Reply:
I wasn't to list it here, but I supposed we could have a policy to expire the jobs and keep them for a while, then delete them after another period.
If we don't pass a policy for a status, then nothing is set for it.